lua/autorun/client/cl_cfc_performance.lua

```diff
@@ -25,63 +25,55 @@ local function removeHooks()
 end
 
 local function runCommands()
-    --RunConsoleCommand("cl_threaded_bone_setup", "1")
-    --RunConsoleCommand("cl_threaded_client_leaf_system", "1")
-    --RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
-    RunConsoleCommand("mat_queue_mode", "-1")
+    RunConsoleCommand("cl_threaded_bone_setup", "1")
+    RunConsoleCommand("cl_threaded_client_leaf_system", "1")
+    RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
```

Comment:
Is this comment for the `RunConsoleCommand` below?

Reply:
It isn't, actually. Another developer told me to set this to `-2` but I couldn't find any documentation that suggested that made any kind of sense. So 🤷‍♂